static/js/portico/help.js

```diff
@@ -78,6 +78,26 @@ function scrollToHash(container) {
     }
 }
 
+function sidebarOverflow() {
+    return ($('.sidebar').prop('scrollHeight') > $('.sidebar').prop('offsetHeight'));
+}
+
+function scrollBarDisplayHide() {
```

Comment:
Probably should be called `scrollBarDisplayUpdate` for clarity, since it shows/hides.